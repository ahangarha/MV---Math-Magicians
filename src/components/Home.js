import React from 'react';

export default function Home() {
  return (
    <>
      <h2>Are you a math magician?</h2>
      <p>
        Well! You might or might not be the one! But here you can do basic
        mathematical operations using a simple calculator.
      </p>
      <p>
        This project is an educational web application made by React to learn
        and practice some key concepts such as the followings:
      </p>
      <ul>
        <li>Create a basic react app</li>
        <li>Work with states and props</li>
        <li>Handle events</li>
        <li>Use router</li>
        <li>Styling in React projects</li>
      </ul>
      <p>
        Use the navigation menu to explore the app. If you like the app, show
        your support by adding a star to the project on&nbsp;
        <a href="https://github.com/ahangarha/MV-MathMagicians/">GitHub</a>
        .
      </p>
    </>
  );
}
