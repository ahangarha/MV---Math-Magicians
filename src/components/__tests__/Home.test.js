import React from 'react';
import Renderer from 'react-test-renderer';
import Home from '../Home';

it('Renders correctly', () => {
  const tree = Renderer
    .create(<Home />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
