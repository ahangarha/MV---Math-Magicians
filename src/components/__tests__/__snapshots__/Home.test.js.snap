// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
Array [
  <h2>
    Are you a math magician?
  </h2>,
  <p>
    Well! You might or might not be the one! But here you can do basic mathematical operations using a simple calculator.
  </p>,
  <p>
    This project is an educational web application made by React to learn and practice some key concepts such as the followings:
  </p>,
  <ul>
    <li>
      Create a basic react app
    </li>
    <li>
      Work with states and props
    </li>
    <li>
      Handle events
    </li>
    <li>
      Use router
    </li>
    <li>
      Styling in React projects
    </li>
  </ul>,
  <p>
    Use the navigation menu to explore the app. If you like the app, show your support by adding a star to the project onÂ 
    <a
      href="https://github.com/ahangarha/MV-MathMagicians/"
    >
      GitHub
    </a>
    .
  </p>,
]
`;
